docs/docs/classifier/sparselr.md

```diff
@@ -0,0 +1,55 @@
+# SparseLogisticRegressionIntentClassifier
```

Comment:
These docs currently won't show up in the sidebar in `mkdocs`. You need to manually add it to the `mkdocs.yml` file.

Reply:
should be fixed now